### PR TITLE
Add Calculator Component Review

### PR DESCRIPTION
**Pull request by Felipeg005**
To achieve project requirements next actions were taken:

- [x] Delete all the boilerplate from CRA (text, images, styles).
- [x] Create a directory called components.
- [x] Inside components create a new Calculator.js file.
- [x] In Calculator.js, create a React component that matches with wireframe
- [x] Don't add any extra design.
- [x] Develop just the UI of the calculator; the math logic to make it work will be added in the next project.
- [x] Use class-based components.
- [x] Once the Calculator is ready, import it and use it in your main component (App.js).
- [x] Test it in the browser; it should render without problems.

[Link to live version](https://felipeg005.github.io/Math-Magicians-React-App/)

Thanks for the review 🤓

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import './App.css';
+import Calculator from './components/calculator';
+
+function App() {
+  const newCalculator = new Calculator();
+  return (
+    newCalculator.render()
+  );
+}
```
Comment (vichuge) [inline src/App.js]: - [x] According to requirements, app.js should be class-bassed too, please change it. 🐱

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import './App.css';
+import Calculator from './components/calculator';
+
+function App() {
+  const newCalculator = new Calculator();
+  return (
+    newCalculator.render()
```
Comment (vichuge) [inline src/App.js]: - [x] Please, the calculator should be called according react components format, on this case, inside return, add `<Calculator />` and you'll have your component inside app.js 🌵

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your PR messages are descriptive ✔️

## Aspects to improve ♻️

- [x] It looks like there's some unnecessary files that you can delete, please do it. 🐶
image


- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (Felipeg005) [reply src/App.js]: Prefect, thanks for remember me that, now is fixed 👍

File: src/App.js (hunk 2 above)
Comment (Felipeg005) [reply src/App.js]: Great, you are right, now I fixed this issue. 😄

**Comment by Davidosky007** [comment]
## Hi @Felipeg005 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**